frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/useInitialAutoLayout.ts

```diff
@@ -26,6 +32,23 @@ const getHiddenNodeIdsFromUrl = async (): Promise<string[]> => {
   return hiddenNodeIds ? hiddenNodeIds.split(',') : []
 }
 
+const getShowModeFromUrl = (): ShowMode => {
+  const urlParams = new URLSearchParams(window.location.search)
+  const showModeQueryParam: QueryParam = 'showMode'
+  const showMode = urlParams.get(showModeQueryParam)
+
+  if (!showMode) {
+    return 'TABLE_NAME'
+  }
+
+  const result = v.safeParse(showModeSchema, showMode)
```

Comment:
Since safeParse can contain any value, it would be simpler to do the following:

```suggestion
  const result = v.safeParse(showModeSchema, showMode)
```

Reply:
Thank you so much🙏! fixed: https://github.com/liam-hq/liam/pull/452/commits/e99b44698436eb30771ba9abe295732711e8d710